### DISCUSSION TITLE
[closed][blueprint][trackfind] Is any version of the FAIRified BLUEPRINT metadata available from TrackHub Registry yet?

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#120] [Team Posts]
@dzerbino @nerdstrike I cannot remember from the Skype call if this was done yet. I cannot find a FAIR BluePrint TrackHub in the TrackHub list (https://www.trackhubregistry.org//api/info/trackhubs), which was the idea, right? We need to make sure that the import mechanism into TrackFind works ASAP! It can be updated with the final metadata corresponding to the last standard changes later. 

If the metadata is available, could you please provide info as how to get them?

**Comment by sveinugu** [comment]
TrackFind issue: https://github.com/elixir-no-nels/trackfind/issues/101

**Comment by nerdstrike** [comment]
We have not deployed to the public server, it is only on the test server as I stated in my email to you on 18/10/2019. If you are happy with the output from the test server, we can consider a full release.

**Comment by sveinugu** [comment]
@nerdstrike Ah. Great! I did not fully register that friday afternoon email in my mind! @dtitov should have what he needs then

**Comment by sveinugu** [comment]
I will keep this discussion point open for any follow-up questions

**Comment by sveinugu** [comment]
Also (sorry for spamming), we can use the test server until the final BLUEPRINT metadata is finalized.
